010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,127 @@
 
-Since Firebase started more than 10 years ago, a new wave of developer-oriented products appeared and the whole serverless wave started. We now have full-stack frameworks like Next.js and platforms like Vercel and Netlify which offer fantastic developer experiences with their git-based workflows. Modern dedicated authentication and identity management services like Clerk, Stytch, or Auth0 are also easily accessible today. As a Serverless Data Platform, Xata fits very well into this growing Jamstack ecosystem.
+At large scale, you pay for the resources you use and you have a high degree of control. For example, for predictable load, you can configure a particular instance size, while for unpredictable load, auto-scaling will likely be more cost-efficient. Similarly, you can choose between standard storage and I/O optimized storage, which is more cost-efficient for I/O-heavy workloads.
 
-## How Xata works
+**Scalability**: Available instance types in Xata go up to 128 vCPUs and 1TB of RAM (for example, `db.r6i.32xlarge`). Storage can be scaled up to 128TiB per cluster. That is quite a lot of headroom for vertical scaling.
 
-Xata is a new type of cloud service: it offers an abstraction layer on top of multiple data stores in order to make application development and operations simpler. We call this type of service a [Serverless Data Platform](/docs/concepts/serverless-data-platform). This document explains the internal architecture of the Xata service, which can be useful to build a correct mental model of Xata. It can also be useful if you want to replicate the same architecture for yourself, which would give you some of the benefits of using Xata.
+When it comes to horizontal scaling, the fact that you can create databases and branches instantly and that you can move them between clusters without downtime, opens up several architectural possibilities. For example, you can create a logical database for each of your customers, and place them in different regions. See the section on [Global control plane for databases and branches](#global-control-plane-for-databases-and-branches) for more details.
 
-If you just want to build apps and use the Xata serverless service, you **do not** need to read this document. We’re solving these problems so that you don’t have to.
+### Shared or dedicated clusters
 
-Xata is a global multi-tenant system, composed of **cells**, each living in a region. At the high level, one Xata **cell** implements the following architecture:
+There are two types of clusters powering the Xata service: shared and dedicated. Database branches can be created on both types of clusters, and they can be moved seamlessly between them, without downtime. It is common to use a dedicated cluster for the production branch and shared clusters for the dev branches.
```

Comment:
As a general comment in this blog, we are talking about dedicated clusters as if they're available to everyone today. Should we at least preface either here or above, that they are an alpha feature? This could be explicit, or just by linking to the dedicated cluster documentation where it is explicit there.

Edit: I see the warning now. It feels too late in the flow, dedicated clusters are mentioned a few times even before this section.

Reply:
I'll cover this in my clarity pass.